HIRS_Utils/src/main/java/hirs/data/persist/certificate/attributes/ComponentIdentifier.java

```diff
@@ -55,11 +56,11 @@
      * Default constructor.
      */
     public ComponentIdentifier() {
-        componentClass = null;
-        componentManufacturer = null;
-        componentModel = null;
-        componentSerial = null;
-        componentRevision = null;
+        componentClass = DeviceInfoReport.NOT_SPECIFIED;
+        componentManufacturer = new DERUTF8String(DeviceInfoReport.NOT_SPECIFIED);
+        componentModel = new DERUTF8String(DeviceInfoReport.NOT_SPECIFIED);
+        componentSerial = new DERUTF8String(DeviceInfoReport.NOT_SPECIFIED);
+        componentRevision = new DERUTF8String(DeviceInfoReport.NOT_SPECIFIED);
         componentManufacturerId = null;
```

Comment:
I wonder if this `componentManufacturerId` should be `componentManufacturerOid` since that's what the type is. Makes it more immediately clear that this isn't a `String` or `Guid` or some kind.

Reply:
I'm not sure what you mean.  At the moment, we don't have an example of what this could end up being.  I hope it does come out as a String.  Don't have a test pattern yet.  The java type is ANS1ObjectIdentifier but it isn't an OID, as in 2.23.133.18.3.1.

It is PrivateEnterpriseNumber, and I don't know what that'll be, so I made it a default value.  It may actually be better to set it to String.

Reply:
Could be, I'll leave that call to you, as I'm not familiar with PrivateEnterpriseNumbers.